CoffeeChat/Controller/HomeViewController.swift

```diff
@@ -94,11 +95,13 @@ class HomeViewController: UIViewController {
     }
 
     private func setupConstraints() {
-        let padding = 35
+        let padding: CGFloat = 70
```

Comment:
I feel 70 is kinda large. To me, the table view for the match summary looks too small because of such a large padding

Reply:
Oh yes, my bad. I changed the value for the nextButton but then forgot to change the value back after I made a new variable for the button padding. I think we should keep this as a TODO? Since this screen probably isn't final on the design side anyways?